src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -411,7 +439,7 @@ private AddTxResult AddCore(Transaction tx, bool isPersistentBroadcast, bool isR
             if (isKnown)
             {
                 // If transaction is a bit older and already known then it may be stored in the persistent storage.
-                Metrics.PendingTransactionsKnown++;
+                Metrics.PendingTransactionsStored++;
```

Comment:
Do we need this? I think PendingTransactionsKnown is fine here

Reply:
I would prefer to use PendingTransactionsKnown only for hashCache checks, which will take place at the beginning of the filtering, to know exactly how many txs we discard before more detailed filtering. Combining number of these txs with number of those already in database will obscure the image imo

Reply:
And phrase "isKnown" is used in code in context of hashCache so I prefer to use it here for hashCache checks and change naming of metric of txs discarded because of being in database